sormas-api/src/main/java/de/symeda/sormas/api/utils/PasswordHelper.java

```diff
@@ -93,7 +94,12 @@ public static String encodePassword(String password, String seed) {
 		}
 	}
 
-	public static String hexEncode(byte[] aData) {
-		return new BigInteger(1, aData).toString(16);
+	public static String hexEncode(byte[] data) {
+
+		StringBuilder hex = new StringBuilder(2 * data.length);
```

Comment:
would suggest to use a char array instead of a string builder here. See https://stackoverflow.com/a/9855338 or implementation of apache commons Hex: https://commons.apache.org/proper/commons-codec/apidocs/src-html/org/apache/commons/codec/binary/Hex.html

Reply:
I replaced the function, but not with the one from commons-codec. ~~We already have it as dependency, but not (yet) in the android-app.~~ I doublechecked, it's already part of the android-app, so I could just use that implementation. @MartinWahnschaffe what's your opinion?

Reply:
If it's already a dependency I'm in favor of using that method.

Reply:
I switched now to commons-codec and removed the extensive testing (don't retest dependencies), I just keep the test that proves the recent problem at hand to be removed.